### DISCUSSION TITLE
Problem with Microsoft Azure credit

### DISCUSSION THREAD

**Discussion by Houayda22** [community/community#61431] [GitHub Education]
### body 
Hello, I have received my student developer pack and I am encountering an issue regarding the microsoft azure credit ,Unfortunately I am unable to utilize microsoft azure for students through the Azure portal  , Can you help me ?

_No response_

**Comment by Akhilrajan1997** [comment]
Same problem, did you get a solution?

**Reply by Houayda22** [reply]
Yes, it is necessary to wait a few days .
